sdk/keyvault/azure-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/X509CertificateProperties.java

```diff
@@ -14,11 +14,11 @@
 class X509CertificateProperties {
```

Comment:
This class properties don't need to have get and set prefixes, because it is a package-private class, is it?

Reply:
Yup, will add in a separate PR.